src/cache/cache_las.c

```diff
@@ -717,6 +717,8 @@ __wt_las_insert_block(WT_CURSOR *cursor,
 				las_value.size = upd->size;
 				break;
 			case WT_UPDATE_BIRTHMARK:
+				WT_ASSERT(session, upd != first_upd || multi->page_las.skew_newest);
+				/* FALLTHROUGH */
```

Comment:
Just sanity checking that we only don't birthmarks at the beginning of a list when `skew_newest = false`, because the sweep server relies on this when discarding data from lookaside.  I have never seen this one fire.